blocks/subheadline-block/index.story.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import SubHeadline from './features/subheadline/default';
+
+export default {
+  title: 'Blocks/Subheadline',
+  parameters: {
+    chromatic: { viewports: [320, 1200] },
+  },
+};
+
+export const subHeadline = () => (<SubHeadline />);
```

Comment:
SubHeadline block can render different content fields, we should have a story for each

Reply:
Updated component to have a root and presentation component for chromatic testing purposes.